src/status_im/contexts/wallet/create_account/new_keypair/check_your_backup/view.cljs

```diff
@@ -62,20 +61,26 @@
 
 (defn- view-internal
   []
-  (let [random-indices  (random-selection)
-        quiz-index      (reagent/atom 0)
-        incorrect-count (reagent/atom 0)
-        show-error?     (reagent/atom false)]
+  (let [random-indices                        (random-selection)
+        quiz-index                            (reagent/atom 0)
+        incorrect-count                       (reagent/atom 0)
+        show-error?                           (reagent/atom false)
+        {:keys [secret-phrase random-phrase]} (rf/sub [:wallet/ui])]
```

Comment:
there seems to be some discussion about pulling in too much data from subs recently. I think this is another case. Can we break this down one step further.
e.g something like `(rf/sub [:wallet/create-account ])` where it's reading from `db :wallet :ui :create-account` or whatever the next key we have is? 
The data model should be cleaned but still, we should segment away the right data to avoid any unexpected/unwanted mixing of data.
cc @ulisesmac

Reply:
Fixed!